config/samples/infra_v1alpha1_clusterscaleprofile.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: infra.nephio.org/v1alpha1
 kind: ClusterScaleProfile
 metadata:
   name: small-fixed-site
+  annotations:
+    config.kubernetes.io/local-config: "binding"
```

Comment:
This is fine for now (won't block checkin). But for controllers that run in the management cluster and inject config, I am using the term "config injector". The "bindings" are related to @justinsb's `bind` kpt function. I wonder if we need to differentiate (e.g., "injection point")? Or perhaps the bind function or config injectors are just different ways to process bindings. WDYT? @justinsb @mortent any thoughts?

Reply:
I don't have a whole lot of context around Justin's "bindings" besides what's in the [README](https://github.com/justinsb/kpt/blob/367a8f452a39357fc5d660214b17fb9d92648235/porch/controllers/klippy/README.md) but it does sound like a different concept, so it could be helpful to use something different here. `config-injector` sounds good to me. While we are on the topic, do you still think that the `config.kubernetes.io/local-config` annotation makes the most sense here?

Reply:
Ok, let's use `config-injection` or `injection-point` (as opposed to "config injector" which is the controller that does the injection).

Yes, `local-config` makes sense because it prevents it from being pushed to the cluster by ConfigSync. At least, I *think* that it will due that as long as it exists and is not explicitly "False". Alternatively we could have a separate annotation, but then people have to remember to add both. Though I suppose there are cases where it is both an injection point and we want it to go to the cluster....hmmm...

How about we go with `config.nephio.org/injection: "True"`? and `local-config` would be orthogonal. We can figure out a different name for the things we build in Porch, but for now let's use this as it is Nephio-specific.